docs/mobile-apps/features.md

```diff
@@ -48,12 +43,11 @@ To initiate a fingerprint or facial recognition action in your live test session
 
 See [Live Mobile App Testing Toolbar](/mobile-apps/live-testing/live-mobile-app-testing/#app-settings) for more information.
 
-
 ### Automated Testing
 
-For automated testing, biometric authentication is supported for simulators, emulators, and iOS real devices. Please note that not all iOS and Android devices support Touch ID or Face ID. Be prepared to handle test cases..
+For automated testing, biometric authentication is supported for simulators, emulators, and iOS and Android real devices. Please note that not all iOS and Android devices support fingerprint or racial recognition, so you may have to adjust your test cases accordingly.
```

Comment:
Just to be sure, are we using fingerprint and face recognition as generic terms here or are we referring to the exact nomenclature of Android and iOS.
If the latter then we need to specific everywhere:
Android -> face recognition, fingerprint
iOS -> Face ID and Touch ID

Reply:
Thanks, @linafarid . I just saw they typo there....guessing "racial" recognition is not a think and might get me in trouble! Eek!

Reply:
I will fix that, but I will defer to you and the experts as to whether we are using generic or specific terms.

Reply:
```suggestion
For automated testing, biometric authentication is supported for simulators, emulators, and iOS and Android real devices. Please note that not all iOS and Android devices support fingerprint or facial recognition, so you may have to adjust your test cases accordingly.
```